daphne/src/lib.rs

```diff
@@ -304,6 +305,23 @@ pub struct DapGlobalConfig {
     pub supported_hpke_kems: Vec<HpkeKemId>,
 }
 
```

Comment:
Reviewer note: This change allows us to reuse some code between daphne and daphne_worker.